<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chooes</class>
 <widget class="QMainWindow" name="Chooes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOption1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 153);
font: 20pt &quot;MS Gothic&quot;;
color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Aws1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOption2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>330</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 153);
font: 20pt &quot;MS Gothic&quot;;
color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Aws2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOption3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 153);
font: 20pt &quot;MS Gothic&quot;;
color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Aws3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOption4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>520</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 153);
font: 20pt &quot;MS Gothic&quot;;
color: rgb(0, 0, 0);
selection-background-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Aws4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bgchoose/bgchoose.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-200</x>
      <y>0</y>
      <width>1221</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/res/backgr.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblQuestion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
text-decoration: underline;
font: 20pt &quot;MS Gothic&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>680</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;MS Gothic&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTimeRemaining">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;MS Gothic&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>time count</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::NoTextInteraction</set>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>btnOption1</zorder>
   <zorder>btnOption2</zorder>
   <zorder>btnOption3</zorder>
   <zorder>btnOption4</zorder>
   <zorder>lblQuestion</zorder>
   <zorder>lblResult</zorder>
   <zorder>lblTimeRemaining</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
